Redefine QMAKE_STRIP so it doesn't strip the ElmerGUI binary.

Index: elmerfem/ElmerGUI/Application/Application.pro
===================================================================
--- elmerfem.orig/ElmerGUI/Application/Application.pro
+++ elmerfem/ElmerGUI/Application/Application.pro
@@ -294,6 +294,9 @@
 win32: RC_FILE += ElmerGUI.rc
 macx: RC_FILE = M3Dicon.icns
 
+# Turn of binary stripping
+QMAKE_STRIP = echo
+
 #------------------------------------------------------------------------------
 # END OF FILE
 #------------------------------------------------------------------------------
